ir/src/main/scala/org/scalajs/core/ir/Utils.scala

```diff
@@ -66,6 +66,35 @@ object Utils {
     // scalastyle:on return
   }
 
+  def printEscapeJS(str: String, out: java.io.Writer): Unit = {
```

Comment:
Actually there is an easy way using a `StringWriter`. I had it on another branch. I'll add it to this one.